### DISCUSSION TITLE
BLYNK not responding keyboard

### DISCUSSION THREAD

**Discussion by l33d** [taligentx/dscKeybusInterface#56] [dscKeybusInterface Discussion]
Hi, 
After 2 months of testing this blynk, i see another problem. Sometimes, after disabling wifi and enabling wifi on my router, the keypad in blynk not responding anymore. It seems linke write pin on dsc is not connected. I must restart esp and works ok. All others function like status zones works flawlessly.

**Comment by l33d** [comment]
Hi, 

Any news in project? :(

**Comment by taligentx** [comment]
I've tested this in the current release and haven't been able to replicate the issue.  After disabling the WiFi access point, Blynk reported the interface as offline and worked normally after re-enabling the WiFi connection.  Please re-open if you're still seeing this on the current release.  Thanks!
